### DISCUSSION TITLE
analysing-my-amazon-purchases/

### DISCUSSION THREAD

**Discussion by giscus** [JakeSteam/blog-personal#26] [Comments]
# analysing-my-amazon-purchases/

Like many people, I purchase… pretty much everything from Amazon, especially since the pandemic. In this post I’ll analyse my own purchasing habits, and provide a template so you can do the same!

https://jakelee.co.uk/analysing-my-amazon-purchases/

**Comment by Jebonicus** [comment]
Wow - thanks for making this. Just tried it on my data. Spreadsheet needed a small bit of updating for 2024. Absolutely shocking how much I've spent! Grand lifetime total of £33k, £7.5k just this year! On what?!? Mostly small-ish value items that I definitely did not need. Jeff has done well out of me this year!

**Reply by JakeSteam** [reply]
It all adds up, right!? I'm a little scared to rerun it a few years later, a house move has definitely increased my stats 😅 Will update the template with newer years.
